src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/SupportedServer.java

```diff
@@ -34,7 +34,8 @@
     , THORNTAIL_V2("thorntail-v2", Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13))
     , LIBERTY("liberty", Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14, MicroProfileVersion.MP20))
     , KUMULUZEE("kumuluzEE", Collections.singletonList(MicroProfileVersion.MP12))
-    , PAYARA_MICRO("payara-micro", Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14, MicroProfileVersion.MP20));
+    , PAYARA_MICRO("payara-micro", Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14, MicroProfileVersion.MP20))
+    , HELIDON("helidon", Collections.singletonList(MicroProfileVersion.MP12));
```

Comment:
I don't think Helidon has passed MP 1.2 TCKs as yet.

Reply:
They don't. That is also the reason why I exclude the example of JWT-auth as they don't support that either.

But I think it is important that MicroProfile Starter has support for all the vendors.

Reply:
I don't have problems to have Helidon listed. I am worried that if the server is displayed when you clicking on MP 1.2, will this send an impression to the end user it fully supports MP 1.2?